docs/cmake-presets.md

```diff
@@ -0,0 +1,272 @@
+|---------|---------|---------|---------|
+|`cmake.useCMakePresets` |  Drive CMake configure, build, and test | `true`, `false`, `auto` | `auto` |
+
+`auto` : evaluates to `true` if there's a `CMakePresets.json` file in the `cmake.sourceDirectory` in the active folder. It evaluates to `false` if there isn't a `CMakePresets.json` file in the `cmake.sourceDirectory` in the active folder. Set `cmake.useCMakePresets` to `true` or `false` to explicitly enable or disable `CMakePresets.json` integration for all CMake projects.
+
+## Configure and build
+
+You can configure and build your CMake project with a series of commands. Open the command palette in Visual Studio Code with `Ctrl+Shift+P`:
+
+![Command pallette: Select Configure Preset](images/command-palette.png)
+
+**CMake: Select Configure Preset** lists the union of non-hidden Configure Presets defined in `CMakePresets.json` and `CMakeUserPresets.json`. Select a Configure Preset to make it the active Configure Preset. This is the `configurePreset` used when CMake is invoked to generate the project build system. The active Configure Preset is displayed in the status bar.
+
+CMake Tools uses the value of `hostOS` in the Microsoft Visual Studio Settings vendor map to hide Configure Presets that don't apply to your platform. See the entry for `hostOS` in the table under [Visual Studio Settings vendor map](cmake-presets-json-reference.md#visual-studio-settings-vendor-map) for more information.
+
+## CMake: configure
+
+To configure the project, run **CMake: Configure** from the command palette. This is the same as running `cmake --preset <configurePreset>` from the command line, where `<configurePreset>` is the name of the active Configure Preset.
+
+## CMake: select Build Preset
+
+**CMake: Select Build Preset** lists the Default Build Preset and the union of non-hidden Build Presets defined in `CMakePresets.json` and `CMakeUserPresets.json`. The Default Build Preset is equivalent to passing `cmake --build` with no additional arguments from the command line. Select a Build Preset to make it the active Build Preset. This is the `buildPreset` used when CMake is invoked to build the project. The active Build Preset is displayed in the status bar.
+
+All Build Presets are required to specify an associated configurePreset. CMake Tools hides Build Presets that don't apply to the active Configure Preset. For more information, see [Build Presets](https://cmake.org/cmake/help/latest/manual/cmake-presets.7.html#build-preset).
+
+## CMake: build
+
+Run **CMake: Build** from the command palette to build the entire project. This is the same as running `cmake --build --preset <buildPreset>` from the command line, where `<buildPreset>` should be replaced by the name of the active Build Preset.
+
+Run **CMake: Build Target** from the command palette to build a single target. You can switch the active build target with **CMake: Set Build Target**.
+
+> [!NOTE]
```

Comment:
GitHub doesn't do any fancy rendering for `[!NOTE]`.  I believe that's a feature of the Microsoft docs pipeline.  We can only use Markdown here, so we will need to emulate what happens in the docs pipeline using just standard Markdown.  "Note" might not actually be needed here since the `>` already sets these "notes" apart.  It's up to you how you want these to appear, but let's just make sure all of them render the same.  Currently I see some capitalization differences.

Reply:
I've updated all instances of [!Note] to be > Note: